detection-rules/impersonation_human_resources.yml

```diff
@@ -8,13 +8,28 @@ source: |
   and regex.icontains(sender.display_name,
                       '(\bh\W?r\W?\b|human resources|hr depart(ment)?|employee relations)'
   )
-  and  (0 < length(body.links) < 10 or length(attachments) > 0)
+  and (0 < length(body.links) < 10 or length(attachments) > 0)
+  
   // Request and Urgency
-  and any(ml.nlu_classifier(body.current_thread.text).entities, .name == "request")
-  and any(ml.nlu_classifier(body.current_thread.text).entities, .name == "urgency")
   and (
-    any(ml.nlu_classifier(body.current_thread.text).intents, .name != "benign")
-    or length(ml.nlu_classifier(body.current_thread.text).intents) == 0  // not benign but not malicious either
+    (
+      any(ml.nlu_classifier(body.current_thread.text).entities,
+          .name == "request"
+          and any(ml.nlu_classifier(body.current_thread.text).entities,
+                  .name == "urgency"
+          )
+      )
+      or any(ml.nlu_classifier(body.current_thread.text).intents,
+             .name in ("cred_theft")
+             and .confidence in ("medium", "high")
+             and (
+               any(ml.nlu_classifier(body.current_thread.text).intents,
+                   .name != "benign"
+               )
+               or length(ml.nlu_classifier(body.current_thread.text).intents) == 0 // not benign but not malicious either
+             )
```

Comment:
what's the intent behind this?

Reply:
Cc @aidenmitchell I think Aiden added that because we saw a null intent, which also doesn't fire !=benign